lib/plugins/packages.js

```diff
@@ -87,6 +87,7 @@ async function refreshPackageInfo (packageName, version) {
   const installedPackageVersion = packageJson && projectPackage.dependencies[packageName]
```

Comment:
@BenSurgisonGDS Might be a good idea to use `.packages[packageName]`

The legacy `dependencies[packageName]` entry is dropped in newer npm versions and only exists for compatibility in older projects created before `"lockfileVersion": 3` was the default

You can use the CLI to switch lockfile version:

```console
npm install --lockfile-version 2
npm install --lockfile-version 3
```

Which will show the legacy `dependencies` no longer exists in v3:

```patch
 {
   "name": "test-prototype",
   "lockfileVersion": 3,
   "packages": {
     "../../../../../../../path/to/govuk-prototype-kit": {
       "version": "13.13.4",
       "dependencies": {}
     }
-  },
-  "dependencies": {
-    "govuk-prototype-kit": {
-      "version": "file:../../../../../../../Users/colin/Sites/GDS/govuk-prototype-kit",
-      "requires": {}
-    }
+  }
 }
```

Reply:
I think it would be great to do this in a future ticket